src/app/products/recurring-deposit-products/view-recurring-deposit-product/view-recurring-deposit-product.component.html

```diff
@@ -0,0 +1,415 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
+  <button mat-raised-button color="primary">
+    <fa-icon icon="edit"></fa-icon>&nbsp;&nbsp;
+    Edit
+  </button>
+</div>
+
+<div class="container">
+
+  <mat-card>
+
+    <mat-card-content>
+
+      <div fxLayout="row wrap" fxLayout.lt-md="column">
+
+        <h2 class="mat-h2" fxFlexFill>{{ recurringDepositProduct.name }}</h2>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <h3 class="mat-h3" fxFlexFill>Details</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill *ngIf="recurringDepositProduct.description">
+          <span fxFlex="40%">Description:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.description }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Short Name:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.shortName }}</span>
+        </div>
+
+        <h3 class="mat-h3" fxFlexFill>Items</h3>
+
+        <mat-divider [inset]="true"></mat-divider>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Currency:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.currency.name }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Decimal Places:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.currency.decimalPlaces }}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Currency in Multiples Of:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.currency.inMultiplesOf | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Minimum Deposit Amount:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.minDepositAmount | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Default Deposit Amount:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.depositAmount | number}}</span>
+        </div>
+
+        <div fxFlexFill>
+          <span fxFlex="40%">Maximum Deposit Amount:</span>
+          <span fxFlex="60%">{{ recurringDepositProduct.maxDepositAmount | number}}</span>
```

Comment:
Some fields might not be mandatory during creation. They should not be displayed if they do not contain a value. Check the API of recurring deposit products and existing code implementation in the community app to know about all those fields and update the code accordingly.

Reply:
I have checked for all the optional and compulsory fields in the API docs and updated accordingly. Please have a look.